SharedResources/en.lproj/Localizable.strings

```diff
@@ -91,11 +91,14 @@
 "fileProvider.onboarding.button.openCryptomator" = "Open Cryptomator";
 
 "keepUnlocked.alert.title" = "Lock Vault?";
-"keepUnlocked.alert.message" = "Your vault is currently unlocked. To enable this option, your vault has to be locked.";
-"keepUnlocked.main.footer.off" = "If you enable this setting, your vault will be guaranteed to be unlocked for and locked after a certain amount of time.";
-"keepUnlocked.main.footer.on" = "If you disable this setting, your vault may lock prematurely when Cryptomator has been terminated by the Files app. Learn more.";
+"keepUnlocked.alert.message" = "This change requires your vault to be locked in order to take effect after your next unlock.";
```

Comment:
```suggestion
"keepUnlocked.alert.message" = "This change requires your vault to be locked in order to take effect.";
```

While technically correct, this additional piece of information doesn't offer any added benefit. Obviously the whole point of this setting will only affect unlocked vaults. 🤓